main.go

```diff
@@ -144,6 +146,11 @@ func sendAlert(data alerttemplate.Alert, getParams url.Values) error {
 		return err
 	}
 
+	mention, err := url.Parse(getParams.Get("mention"))
```

Comment:
I'd call this variable `mentionAll` to give a better idea of what it's referring to.